### PR TITLE
Allow setting deployment replica count

### PR DESCRIPTION
**Pull request by Robbilie**
In by its own there will probably be not that many people that have a usecase for this, i dont think it hurts to have it configurable though, keeps me from making a hard fork of this chart for now :P

### CODE REVIEW

File: charts/minecraft/values.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@ image: itzg/minecraft-server
 imageTag: latest
 imagePullSecret: ""
 
+replicaCount: 1
```
Comment (itzg) [inline charts/minecraft/values.yaml:6]: It feels like this needs a big warning similar to what you described at the top of the PR.

Comment (Robbilie) [reply charts/minecraft/values.yaml:6]: you think a comment in the values.yaml is enough? ill add one

Comment (Robbilie) [reply charts/minecraft/values.yaml:9]: there we go

Comment (itzg) [reply charts/minecraft/values.yaml:9]: Thanks!

**Review by itzg — APPROVED** [review]
Enjoy!

**Comment by Robbilie** [comment]
i got one more thing coming :D

**Comment by Robbilie** [comment]
https://github.com/itzg/minecraft-server-charts/pull/99

**Comment by ChipWolf** [comment]
As a point of interest @Robbilie, why did you add this?

**Comment by Robbilie** [comment]
@ChipWolf I mentioned it in the other PR

https://github.com/itzg/minecraft-server-charts/pull/97

I wanna run a synchronization sidecar, that's why I added some features to the chart, got another one coming at some point for podlabels

**Pull request merged**